docs/components/segmentedBar.md

```diff
@@ -0,0 +1,66 @@
+
+# SegmentedBar
```

Comment:
please use documentation template to be consistent with other pages

Reply:
Template is fixed based on https://github.com/vuexp/vuexp/blob/development/docs/components/_template.md